build-rr.sh

```diff
@@ -374,7 +374,7 @@ buildrump ()
 		TLSCFLAGS="-F CFLAGS=-DRR_USE_TLS"
 	else
 		CURLWP_METHOD=hypercall
-		TLSCFLAGS="-F CFLAGS=-D_PTHREAD_GETTCB_EXT=_lwp_get_tls_tcb"
+		TLSCFLAGS="-F CFLAGS=-D_PTHREAD_GETTCB_EXT=_lwp_get_tls_tcb -F CFLAGS=-fno-stack-protector"
```

Comment:
This is on the dangerous side, what is the reason for this?

Reply:
Copying conversation:
@ricarkol :
it uses TLS
and we don't support it
we are planning to add a set_tls function in solo5

@lumjjb: 

Let's add an issue for this.